docs/installation/generic.md

```diff
@@ -153,11 +170,18 @@ export REDIS_URL=redis://localhost:6379/0
 ```bash
 celery worker -A app.celery &
 ```
-
+### Note: The '&' in the above command means detaching from the console. To avoid use the following command instead
+```bash
+celery worker -A app.celery --loglevel=INFO
+```
 ##### Starting the app generator web server
 ```bash
 gunicorn -b 0.0.0.0:8080 app:app --enable-stdio-inheritance --log-level "info"
 ```
 
 You can now visit [http://localhost:8080](http://localhost:8080/) on your browser to access the app generator.
 
+### A FEW PRECAUTIONS/TIPS:
+### Run Celery and Gunicorn command on different commandline windows. 
```

Comment:
Also your headings are bigger than other headings, yours should be smaller as they are sub sections